juddges/data/utils.py

```diff
@@ -0,0 +1,26 @@
+from datetime import datetime
+from pathlib import Path
+from typing import Any, Dict, Generator, Iterable
+
+from jsonlines import jsonlines
+
+
+def save_jsonl(records: Iterable[Dict[str, Any]], out: Path | str, mode="w") -> None:
```

Comment:
check out srsly lib https://github.com/explosion/srsly

Reply:
`srsly` does not support iterative writing nor reading forcing user to keep the all data in memory